repositories/src/main/scala/cromwell/pipeline/datastorage/commit/CreateFileRequest.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.pipeline.datastorage.commit
```

Comment:
Are you sure it's a good directory to have this file in?

Also, maybe we should have one object for all entities related to `File`?
So we will not be forced to create a lot of files with 10 lines.

Reply:
Ok

Reply:
Will create File object with all case classes for requests